CMakeLists.txt

```diff
@@ -5,18 +5,21 @@ project(tracelogging)
 include(GNUInstallDirs)
 
 add_compile_options(-Wall -Wextra -Werror)
+set(BUILD_TESTS ON CACHE BOOL "Build tests.")
 
 # Include source
 add_subdirectory(src)
 
 # Only include testing stuff if we are the top level
-if (${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME})
-    enable_testing()
+if(BUILD_TESTS)
```

Comment:
Please name this 'TRACELOGGING_BUILD_TESTS'

Reply:
done